static/src/javascripts/projects/common/modules/navigation/newSticky.js

```diff
@@ -107,48 +113,36 @@ define([
     };
 
     var render = function (els, state) {
-        // Reset so we have a clean slate
-        els.$header.css({
-            'transition': '',
-            'margin-top': ''
-        });
-        els.$adBanner.css({
-            'position': '',
-            'top': '',
-            'max-height': '',
-            'overflow': '',
-            'transition': ''
-        });
-
-        // Set
         // TODO: move into stylesheets when productionised
         els.$document.addClass('new-sticky-ad');
-        els.$header.css({ 'margin-top': state.adHeight });
-        els.$adBanner.css({ 'max-height': state.adHeight });
+
+        var transitionTimingFunction = 'cubic-bezier(0, 0, 0, .985)';
```

Comment:
We've just copied this from what we had with the old sticky nav: https://github.com/guardian/frontend/blob/master/static/src/stylesheets/module/_adslot.scss#L120

It looks like this: 
![image](https://cloud.githubusercontent.com/assets/921609/12950029/08ad1f52-d003-11e5-8e85-a9379ea62585.png)

@stephanfowler and I did think about changing it to ease-in